frontend/src/components/AddMember/index.js

```diff
@@ -1,168 +1,243 @@
-          <Hero
-            title="Creează cont pentru un membru al familiei"
-            subtitle="Înregistrează un membru al familiei care nu poate accesa platforma și ajută-l să își facă evaluarea zilnică a simptomelor COVID-19."
-          />
-          <Input label="Nume și prenume" type="text" required={true} />
-          <div className="columns">
-            <div className="column is-3">
-              <Input
-                label="Vârstă"
-                type="text"
-                onChange={validateAge}
-                required={true}
-              />
-            </div>
-            <div className="column is-3">
-              <Select
-                label="Gen"
-                options={options.gender}
-                selectProps={{
-                  onChange: toggleSex,
-                  className: "is-extended"
-                }}
-              />
-            </div>
-          </div>
-          <div className="columns">
-            <div className="column is-6">
-              <Select
-                label="Legătura familiala"
-                options={
-                  female
-                    ? options.relationship.female
-                    : options.relationship.male
-                }
-              />
-            </div>
-          </div>
-          <RadioList
-            label="Are condiții de sănătate preexistente?"
-            description="Spune-ne dacă suferă de anumte boli cronice, diabet, hipertensiune, etc."
-            type="horizontal"
-            options={options.yesNo}
-            onChange={togglePreexistingConditions}
-            required={true}
-          />
-          {preexistingConditions && (
-            <div className="column is-6 is-multiple">
-              <Select
-                label="Condiții de sănătate preexistente"
-                options={options.preexistingConditions}
-                selectProps={{ multiple: "multiple" }}
-              />
-            </div>
-          )}
-          <RadioList
-            label="Are anumite dizabilități?"
-            description="Spune-ne dacă sferă de anumite dizabilitati Ex locomotorii, mentale, de vorbire, etc."
-            type="horizontal"
-            options={options.yesNo}
-            onChange={toggleDisabilities}
-          />
-          {disabilities && (
-            <Input label="Dizabilități" type="text" required={true} />
-          )}
-          <RadioList
-            label="În ultima perioadă a fost în:"
-            type="horizontal"
-            options={options.isolation}
-          />
-
-          <div className="columns">
-            <div className="column is-4 is-offset-8">
-              <Button type="primary" size="medium" inputType="submit">
-                Adaugă
-              </Button>
-            </div>
-          </div>
-        </div>
-      </form>
-    </SidebarLayout>
+    <form className="user-profile-form">
+      {steps[currentStep]}
+      <Button type="button" onClick={nextStepHandler}>
```

Comment:
[Unfortunately](https://github.com/code4romania/taskforce-fe-components/issues/149), the `type` prop on our `Button` component is not what you assume :(.

Reply:
```suggestion
      <Button inputType="button" onClick={nextStepHandler}>
```